### PR TITLE
Frame Dumping: Fix window-size framedumping

### PR DESCRIPTION
**Pull request by stenzek**
Broken as of hybrid XFB.

Also changes the GUI string from "Full" resolution to "Internal" resolution, as I think that is clearer. But feel free to correct me here.

### CODE REVIEW

**Comment by iwubcode** [comment]
This was actually intentional, I had wanted to get away from alternate forms of frame dumping.  I guess the code here isn't too bad though.

**Comment by RisingFog** [comment]
Window size dumping is useful in certain scenarios, so I feel like keeping it is a good idea.

**Comment by stenzek** [comment]
Main case I see for it is screenshots, if you're using high IRs, screenshots are *very* slow to create.

Also the option was still in the UI ;)

**Comment by iwubcode** [comment]
Yup, this is so little code, if we want to keep the window-size dumping, sounds good to me.  LGTM

File: Source/Core/DolphinWX/VideoConfigDiag.cpp [hunk 1]
```diff
@@ -843,7 +843,7 @@ VideoConfigDiag::VideoConfigDiag(wxWindow* parent, const std::string& title)
                                             Config::GFX_CACHE_HIRES_TEXTURES);
       szr_utility->Add(cache_hires_textures);
 
-      szr_utility->Add(CreateCheckBox(page_advanced, _("Full Resolution Frame Dumps"),
+      szr_utility->Add(CreateCheckBox(page_advanced, _("Internal Resolution Frame Dumps"),
```
Comment (leoetlino) [inline Source/Core/DolphinWX/VideoConfigDiag.cpp]: This should be changed in DolphinQt2 too:
https://github.com/dolphin-emu/dolphin/blob/6cfd7ba/Source/Core/DolphinQt2/Config/Graphics/AdvancedWidget.cpp#L65

**Comment by stenzek** [comment]
@leoetlino Good catch, thanks.

**Pull request merged**